### PR TITLE
Manage climbing_outdoor_type during migration

### PR DESCRIPTION
**Pull request by fbunoz**
See https://github.com/c2corg/v6_api/issues/474 : 
During migration, all routes whose "activities" field has "rock_climbing" AND has not "hiking", the field "climbing_outdoor_type" must be set to "multi".

I change the route migration script to manage the climbing_outdoor_type value about the activities.
This is just a proposal, I didn't tested the script.

### CODE REVIEW

**Comment by asaunier** [comment]
@fbunoz could you make the change suggested by @tsauerwein (I also find it more readable).

In addition the tests report the following coding errors:
> c2corg_api/scripts/migration/documents/routes.py:84:74: E203 whitespace before ':'
c2corg_api/scripts/migration/documents/routes.py:86:1: W293 blank line contains whitespace

**Comment by asaunier** [comment]
Please squash your commits (only one commit).

File: c2corg_api/scripts/migration/documents/routes.py [hunk 1]
```diff
@@ -80,6 +80,10 @@ def get_document(self, document_in, version):
             # (in v6 activities are required)
             activities = ['skitouring']
 
+        climbing_outdoor_type = None
+        if 'rock_climbing' in activities and 'hiking' not in activities:
+            climbing_outdoor_type = 'multi'
```
Comment (fbunoz) [inline c2corg_api/scripts/migration/documents/routes.py]: No, we talk about a route field : 
https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/route.py#L147-L148

Comment (tsauerwein) [inline c2corg_api/scripts/migration/documents/routes.py]: Sorry, my mistake.

**Comment by tsauerwein** [comment]
Thanks!

**Comment by asaunier** [comment]
Thanks @fbunoz your help is very much appreciated!

**Pull request merged**